### DISCUSSION TITLE
SLANG?

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#47] [Generale]
slang? 

Guardatevi sto tipo, si è ovviamente fatto almeno un paio di birre, ma è divertente.
In pratica spiega agli altri americani, come parlano loro in L.A e in california.
molto probabile che lo slang sia nato nel periodo del nonno del tizio in video. fine anni 50 inizi 60.
ma infatti già ai tempi di Charles Bukowski si parlava già così.

https://www.youtube.com/watch?v=-VD3cC8LeWQ&t=155s

quest'altro video serio, parlano dei differenti dialetti in california

https://www.youtube.com/watch?v=80xMoaAiHfI&t=40s

**Comment by tesivo** [comment]
Tutorial per americani per far capire come gli americani di LA parlano americano hihi
